hack/utils/checklegacy.sh

```diff
@@ -0,0 +1,23 @@
+#! /bin/bash
```

Comment:
Advise moving to  https://github.com/kubevela/kubevela.io/tree/main/static/script

Reply:
The usage:
```
curl -fsSl https://kubevela.net/script/checklegacy.sh | bash
```

Reply:
Can't agree more